components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConsumerImpl.java

```diff
@@ -3906,6 +3936,11 @@ else if (APIConstants.API_KEY_TYPE_SANDBOX.equals(tokenType)) { // if
             APIUtil.logAuditMessage(APIConstants.AuditLogConstants.APPLICATION, appLogObject.toString(),
                     APIConstants.AuditLogConstants.UPDATED, this.username);
 
+            ApplicationRegistrationEvent applicationRegistrationEvent = new ApplicationRegistrationEvent(
```

Comment:
this event has to go to completeApplicationRegistration

Reply:
resolved